#the header of the message
Header header

int32 vert_sign